packages/tools/pactjs-cli/src/retrieve-contract/tests/retrieve-contract.test.ts

```diff
@@ -1,5 +1,6 @@
 jest.mock('fs');
 jest.mock('../../utils/retrieveContractFromChain');
+const path = require('path');
```

Comment:
This looks like two different ways to imports things, is there a reason to deviate from the rest in this file?

Reply:
Nice catch!
No, I've just updated it